test/router/V4Router.t.sol

```diff
@@ -282,7 +282,7 @@ contract V4RouterTest is RoutingTestHelpers {
 
         plan = plan.add(Actions.SETTLE, abi.encode(key0.currency0, ActionConstants.CONTRACT_BALANCE, false));
         plan = plan.add(Actions.SWAP_EXACT_IN_SINGLE, abi.encode(params));
-        plan = plan.add(Actions.TAKE_ALL, abi.encode(key0.currency1, address(this)));
+        plan = plan.add(Actions.TAKE_ALL, abi.encode(key0.currency1, 0));
```

Comment:
can we add a constant to `LiquidityOperators` similar to `MAX_SLIPPAGE_INCREASE` so this isnt a magic number and is more readable

`MIN_SLIPPAGE_TAKE`?